utils_cv/detection/dataset.py

```diff
@@ -34,7 +35,7 @@ def get_transform(train: bool) -> List[object]:
     """
     transforms = []
     transforms.append(ToTensor())
-    if train:
+    if train is True:
```

Comment:
[minor] do we want the "is True" here?

Reply:
oh yeah i was trying many things to test, forgot to remove this.

Reply:
done